tools/clang/test/CodeGenHLSL/batch/passes/globalopt/entrypoint_name_invariant_main.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -T vs_6_0 -E main %s | FileCheck %s
```

Comment:
If I get a chance, I'll implement support for multiple RUN lines, this is a perfect use case.